Cargo.toml

```diff
@@ -95,3 +96,4 @@ winapi = "0.3.9"
 [profile.release]
 opt-level = 3
 panic = "abort"
+debug = true
```

Comment:
Is this to output debug symbols?

Reply:
yes, correct!

later on, though, when I build the binary I strip out the debug information (see the CI job) so it won't affect the final binary size.